### DISCUSSION TITLE
How to configure tokenUrl in swagger authorize?

### DISCUSSION THREAD

**Discussion by Krupskis** [fastapi/fastapi#10261] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI()

app = FastAPI()

@app.get("/")
def get():
  return {"hello": "world"}
```

### Description

I am using a custom auth, which has a path of `api/v1/auth/login` as a tokenUrl. Is there a way for OPENAPI config to use this new url?
<img width="449" alt="Screenshot 2023-09-18 at 13 13 05" src="https://github.com/tiangolo/fastapi/assets/44723913/f830e412-b807-4142-aa9a-90395433fc99">

https://github.com/tiangolo/fastapi/issues/4328 This is a related discussion, but it didn't seem to solve it. Trying

```
oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/asd/token")
```

Does not change the token url.

### Operating System

macOS

### FastAPI Version

0.103.1

### Pydantic Version

2.3.0

### Python Version

3.10.12

**Comment by nemosanima** [comment]
Hi!

Try to specify without a slash at the beginning:
```python
oauth2_scheme = OAuth2PasswordBearer(tokenUrl='api/v1/auth/login')
```
